de.tubs.variantsync.core/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,4 +1,14 @@
 eclipse.preferences.version=1
```

Comment:
@h3ssto Is it usual for eclipse projects to submit the .settings directory to vcs? Should it be removed?

Reply:
No it is not common and yes it should not be part of the repo.

Reply:
Ok, so please remove this folder and add it to the gitignores please @tcerny @jeremiaheinle @wurstbroteater .